test/test_kernel.py

```diff
@@ -26,3 +21,34 @@ def test_kernel_commands(self):
                     kernel.console(command.split("/")[-1] + "\n")
         finally:
             kernel.shutdown()
+
+class TestGetSafePath(unittest.TestCase):
+    def test_get_safe_path(self):
+        from meerk40t.kernel import get_safe_path
+        import os
+        """
+        Tests the get_safe_path method for all o/ses
+        """
+        sep = os.sep
+        self.assertEquals(
+            str(get_safe_path("test", system="Darwin")),
+            (
+                os.path.expanduser("~")
+                + sep + "Library" + sep + "Application Support" + sep + "test"
+            )
+        )
+        self.assertEquals(
+            str(get_safe_path("test", system="Windows")),
+            (
+                os.path.expandvars("%LOCALAPPDATA%")
+                + sep + "test"
+            )
+        )
+        self.assertEquals(
+            str(get_safe_path("test", system="Linux")),
+            (
+                os.path.expanduser("~")
+                + sep + ".config" + sep + "test"
+            )
+        )
```

Comment:
I'm not sure this would work. Since it would only apply to one OS.

Reply:
This tests all o/s cases (except something unknown).